README.md

```diff
@@ -80,9 +81,9 @@ On Plone 6, we recommend using the known good set (KGS) of package versions that
 
 On Plone 5, Volto is currently tested with the following packages pinned to specific versions, and we recommend using the same versions, which are:
```

Comment:
@sneridagh The versions we give here need to match the versions in KGS5 in the Makefile, if we want to make the claim that that's what is tested. Currently they do not match.